src/h3lib/lib/algos.c

```diff
@@ -1142,7 +1142,7 @@ H3Error H3_EXPORT(cellsToLinkedMultiPolygon)(const H3Index *h3Set,
     }
     _vertexGraphToLinkedGeo(&graph, out);
     if (normalizeMultiPolygon(out)) {
-        return E_FAILED;
+        return E_FAILED; // Q: normalizeMultiPolygon returns error codes, should we forward those to the public API?
```

Comment:
Yes, I assume that would be better. It looks like these didn't get rolled into the generalized error enum, so we'd need to move them there. It also looks like at least `normalizeMultiPolygon` and maybe some others return `int` instead of `H3Error`, which we should also update.

Reply:
Moved discussion to https://github.com/uber/h3/issues/628